### ISSUE TITLE
Error: index is out of range

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pyright#1139]
**Describe the bug**
output shows 

```
Error: index is out of range
    at TextRangeCollection.getItemAt 
    at Parser._peekToken 
    at Parser._makeExpressionOrTuple 
    at Parser.parseTextExpression
    at Parser._parseFormatStringSegment 
    at Parser._parseFormatString 
    at Parser._parseStringList 
    at Parser._parseAtom 
    at Parser._parseAtomExpression 
    at Parser._parseArithmeticFactor 
```

**To Reproduce**
paste this to empty file
```python
msg = "test"

a = f"{}"
```

**Expected behavior**
no error on output window

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/1333179/98165706-343b5580-1e9b-11eb-94e3-8815fe221451.png)

**VS Code extension or command-line**
extension

**Comment by heejaechang** [comment]
found bug this line (https://github.com/microsoft/pyright/blob/master/packages/pyright-internal/src/parser/parser.ts#L3429) is missing "return"

**Comment by jakebailey** [comment]
Will fix in pylance and sync back before our release.

**Comment by erictraut** [comment]
I think you've found the oldest bug in the code base. This is one of the first functions I wrote in pyright. :)

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.84, which I just published. It was already included in the last release of Pylance.
